release/Makefile

```diff
@@ -139,8 +139,9 @@ base.txz:
 	    -s ${WORLDDIR} -d "${.OBJDIR}/${DISTDIR}/base/var/db/etcupdate" \
 	    -L /dev/null -N
 	echo "./var/db/etcupdate type=dir uname=root gname=wheel mode=0755" >> ${.OBJDIR}/${DISTDIR}/base.meta
-	sed -n 's,^\.,./var/db/etcupdate/current,p' ${.OBJDIR}/${DISTDIR}/base/var/db/etcupdate/current/METALOG \
-	    >> ${.OBJDIR}/${DISTDIR}/base.meta
+	sed -n 's,^\.,./var/db/etcupdate/current,p' \
+	    ${.OBJDIR}/${DISTDIR}/base/var/db/etcupdate/current/METALOG | \
+	    ${METALOG_SORT_CMD} >> ${.OBJDIR}/${DISTDIR}/base.meta
```

Comment:
Wait, where is `METALOG_SORT_CMD` defined for relelase/* Makefiles?

Reply:
oof... it is not.

And the reason this "worked" for removing the non-determinism is because `echo "foo" | > file` is valid shell, and results in empty `file`. So the etc entries aren't being sorted... they're just not being added at all.

Reply:
fixed in 9da5a0f1d057 and confirmed it packages the same on three systems

Reply:
You still have to do what METALOG_SORT_CMD does or this won't be sound on systems with LC_COLLATE!=C.

Reply:
Updated in 0d3b7348